bert_tokenizer.py

```diff
@@ -0,0 +1,122 @@
+"""
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import csv
+import os
+import random
+import numpy as np
+import pandas as pd
+import bert
+
+from sklearn.model_selection import train_test_split
+
```

Comment:
I think it makes sense to talk about the overall structure of NYT data and the edges cases you have handled below by adding a comment section here about data preprocessing